Operational Technology (IOT)/Reporting & Monitoring

```diff
@@ -0,0 +1,61 @@
+Reporting and Monitoring
+========================
+
+In Cyberphysical systems (CPS), there is always a blending of data and physical
+activity and physically adjacent systems. In a trivial example, a pedestrian
+stepping in front of an autonomous car may be a data event the sets an alarm and
+a trigger. How fast the car can brake is a physical activity that is always
+finite in duration, but that may be different on a wet road or an icy one. In
+any case, stopping faster than the car behind can do may also result in
+cataclysmic failure.
+
+On a power transmission grid, blocking the software that operates a substation
+with a DDOS Attack may be indistinguishable in effect from simply shooting a
+transformer with a rifle. CPS cybersecurity must always consider the blending of
+cyber and physical.
+
+A CPS system, whether a microgrid or factory floor or a wet utility or a swarm
+of drones, likely consists of multiple autonomous or semiautonomous systems.
+Cybersecurity monitoring if such systems may include monitoring the “other”
```

Comment:
In "... monitoring if such systems ...", "if" should be "of"

Reply:
Agrred